Ports/VVVVVV/package.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=VVVVVV
+version=2.3.6
+useconfigure=true
+auth_type=sha256
+files="https://github.com/TerryCavanagh/VVVVVV/archive/refs/tags/${version}.tar.gz VVVVVV-${version}.tar.gz a3366aab9e8462d330044ab1ec63927e9f5c3801c0ed96b24f08c553dcb911e9"
+workdir="${port}-${version}"
+configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt" "-DCMAKE_BUILD_TYPE=Release" "-Sdesktop_version")
+depends=("SDL2" "SDL2_mixer")
+icon_file="desktop_version/icon.ico"
+launcher_name="VVVVVV"
+launcher_category="Games"
+launcher_command="/opt/VVVVVV/VVVVVV"
+
+pre_patch() {
+    # Convert to Unix line endings for patches to apply
+    run sed -i -e 's,'$'\015'',,' third_party/tinyxml2/tinyxml2.cpp
```

Comment:
As far as I know, our patches should have the ability to contain CRLF line endlings: c0c72c6b91e5b071c05c64ebfa4808c79bec2a33

Reply:
Ok, the problem here is that the patches generated by `git format-patch` are having LF line endings, while the original file has CRLF. The `sed` invocation here is to force the target file to have LF endings for the patch to apply.

Is there a way for `git format-patch` to replace LF with CRLF on the exported patch files? If yes, then we can get rid of the `sed` trickery here.